_apps/triage-new-issues.md

```diff
@@ -0,0 +1,63 @@
+---
+# A human-friendly name of your listing
+title: Triage New Issues
+
+# A short description of what your app does
+description: Adds `triage` label to newly-created issues and then removes it when other label(s) are added
+
+# The slug of your hosted app on GitHub (https://github.com/apps/YOUR-SLUG)
+slug: triage-new-issues
+
+# Include a few screenshots that show your app in action
+screenshots:
+- https://i.imgur.com/bdbDaPz.png
+- https://i.imgur.com/HMndYuP.png
+- https://i.imgur.com/fOp0Est.png
+- https://i.imgur.com/ldrHDzZ.png
+
+# The GitHub usernames of anyone who authored the app
+authors: [ tunnckoCore, olstenlarck ]
+
+# The repository where the code is located
+repository: tunnckoCore/triage-new-issues
+
+# The address where this app is deployed
+host: https://triage-new-issues.now.sh
+---
+
+
+## Use programmatically
+
+Install it using [**yarn**](https://yarnpkg.com) **v1** or above / [**npm**](https://www.npmjs.com) **v5** or above.
+
+```
+$ yarn add {%= name %}
```

Comment:
Looks like you need to fill in the name of the package on npm or remove this section of the readme — `{%= name %}` isn’t valid Liquid.

Reply:
ah, yea. Thanks.